composer.json

```diff
@@ -34,6 +34,10 @@
     {
       "type": "git",
       "url": "https://github.com/totten/paratest.git"
+    },
+    {
+      "type": "git",
+      "url": "https://github.com/eileenmcnaughton/php-symbol-diff.git"
```

Comment:
Thanks for tracing this. I think it's resolved after committing https://github.com/totten/php-symbol-diff/commit/8d1a1e31867275583c75c15896b6a8cf0b540439